### ISSUE TITLE
Add new Attendance entry manually

### ISSUE THREAD

**Issue by vicenteguardiola** [dhanasingh/redmine_wktime#88]
Is there any way an  admin user could add a new Attendance Entry manually?

As far as I could find, an admin can edit attendance entries and add a new one when there is no entry for a given day. However, I don't see any option to add a new entry to a day which already has some entry.

Thanks

**Comment by karthickmadhiazhagan** [comment]
No, I think there is no way probably. We will take this as feature request. 
We will put it in the future release.

Thanks,
Karthick
